<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCS Video Player</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
    <item>
     <widget class="QVideoWidget" name="videoWidget" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottomLayout" stretch="0,0,0,0,0,1,0,0">
      <item>
       <widget class="QPushButton" name="previousButton">
        <property name="toolTip">
         <string>Previous</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::MediaSkipBackward"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="seekBackwardButton">
        <property name="toolTip">
         <string>Skip backward</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::MediaSeekBackward"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playPauseButton">
        <property name="toolTip">
         <string>Play</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="seekForwardButton">
        <property name="toolTip">
         <string>Skip forward</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::MediaSeekForward"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextButton">
        <property name="toolTip">
         <string>Next</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::MediaSkipForward"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="playingSlider">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="muteButton">
        <property name="toolTip">
         <string>Mute</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::AudioVolumeHigh"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="volumeSlider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1098</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Url"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFull_screen"/>
    <addaction name="actionHide_bottom_panel"/>
   </widget>
   <widget class="QMenu" name="menuPlayback">
    <property name="title">
     <string>Playback</string>
    </property>
    <addaction name="actionPlayPause"/>
    <addaction name="actionSkip_forward"/>
    <addaction name="actionSkip_backward"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
   </widget>
   <widget class="QMenu" name="menuVideo">
    <property name="title">
     <string>Video</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSubtitles">
    <property name="title">
     <string>Subtitles</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSound">
    <property name="title">
     <string>Sound</string>
    </property>
    <addaction name="actionMute"/>
    <addaction name="actionIncrease_volume"/>
    <addaction name="actionDecrease_volume"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_PCS_Video_Player"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuPlayback"/>
   <addaction name="menuVideo"/>
   <addaction name="menuSound"/>
   <addaction name="menuSubtitles"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionFull_screen">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ViewFullscreen"/>
   </property>
   <property name="text">
    <string>Full screen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionPlayPause">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionSkip_forward">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaSeekForward"/>
   </property>
   <property name="text">
    <string>Skip forward</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionSkip_backward">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaSeekBackward"/>
   </property>
   <property name="text">
    <string>Skip backward</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaSkipForward"/>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaSkipBackward"/>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </action>
  <action name="actionMute">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::AudioVolumeHigh"/>
   </property>
   <property name="text">
    <string>Mute</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="actionIncrease_volume">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::AudioVolumeMedium"/>
   </property>
   <property name="text">
    <string>Increase volume</string>
   </property>
   <property name="shortcut">
    <string>0</string>
   </property>
  </action>
  <action name="actionDecrease_volume">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::AudioVolumeLow"/>
   </property>
   <property name="text">
    <string>Decrease volume</string>
   </property>
   <property name="shortcut">
    <string>9</string>
   </property>
  </action>
  <action name="actionOpen_Url">
   <property name="text">
    <string>Open Url</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_PCS_Video_Player">
   <property name="text">
    <string>About PCS Video Player</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionHide_bottom_panel">
   <property name="text">
    <string>Hide bottom panel</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
